### PR TITLE
Translation of reference-javascript-environment-requirements

### PR DESCRIPTION
**Pull request by fjoshuajr**

### CODE REVIEW

File: content/docs/reference-javascript-environment-requirements.md [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
 
-React 16 depends on the collection types [Map](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map) and [Set](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set). If you support older browsers and devices which may not yet provide these natively (e.g. IE < 11) or which have non-compliant implementations (e.g. IE 11), consider including a global polyfill in your bundled application, such as [core-js](https://github.com/zloirock/core-js) or [babel-polyfill](https://babeljs.io/docs/usage/polyfill/).
+O React 16 depende dos tipos de colecção [Map](https://developer.mozilla.org/pt-BR/docs/Web/JavaScript/Reference/Global_Objects/Map) e [Set](https://developer.mozilla.org/pt-BR/docs/Web/JavaScript/Reference/Global_Objects/Set). Se precisas manter suporte para navegadores antigos e dispositivos que podem ainda não ter estes recursos de forma nativa (ex: IE < 11) ou tenham uma implementação que não seja padrão (ex. IE 11), considera a inclusão de um _polyfill_ global no _bundle_ da tua aplicação, tal como o pacote [core-js](https://github.com/zloirock/core-js) ou [babel-polyfill](https://babeljs.io/docs/usage/polyfill/).
```
Comment (jbarradas) [inline content/docs/reference-javascript-environment-requirements.md]:
```suggestion
O React 16 depende dos tipos de colecção [Map](https://developer.mozilla.org/pt-BR/docs/Web/JavaScript/Reference/Global_Objects/Map) e [Set](https://developer.mozilla.org/pt-BR/docs/Web/JavaScript/Reference/Global_Objects/Set). Para suportar navegadores antigos e dispositivos que ainda não tenham estes tipos de forma nativa (ex: IE < 11) ou tenham uma implementação incompatível (ex. IE 11), considera a inclusão de um _polyfill_ global no _bundle_ da tua aplicação, tal como o pacote [core-js](https://github.com/zloirock/core-js) ou [babel-polyfill](https://babeljs.io/docs/usage/polyfill/).
```

File: content/docs/reference-javascript-environment-requirements.md [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
 
-A polyfilled environment for React 16 using core-js to support older browsers might look like:
+Um ambiente com _polyfill_ para o React 16, que usa core-js para dar suporte à navegadores antigos é mais ou menos assim:
```
Comment (jbarradas) [inline content/docs/reference-javascript-environment-requirements.md]:
```suggestion
Um ambiente com _polyfill_ para o React 16, que usa core-js para dar suporte a navegadores antigos é mais ou menos assim:
```

**Review by jbarradas — CHANGES_REQUESTED** [review]

**Comment by fjoshuajr** [comment]
@jbarradas feedback addressed 🙂

**Review by jbarradas — APPROVED** [review]

**Review by 0xtelmo — APPROVED** [review]

**Comment by fjoshuajr** [comment]
🚀

**Pull request merged**